e2e/src/simple-token-transfer.spec.ts

```diff
@@ -0,0 +1,172 @@
+const {Assertion, expect} = require("chai");
```

Comment:
What about creating a new `tests` folder and move `e2e` there (in addition to per-project unit tests we'd add soon)? We're very (very) soon have other "global" test tools and projects there.

Reply:
@electricmonk WDYT?